src/libs/utils.js

```diff
@@ -192,6 +193,7 @@ export const setUserRoleStatuses = (user, Storage) => {
   user.isAlumni = currentUserRoles.indexOf(USER_ROLES.alumni) > -1;
   user.isSigningOfficial = currentUserRoles.indexOf(USER_ROLES.signingOfficial) > -1;
   user.isDataSubmitter = currentUserRoles.indexOf(USER_ROLES.dataSubmitter) > -1;
+  user.isServiceAccount = currentUserRoles.indexOf(USER_ROLES.serviceAccount) > -1;
```

Comment:
I guess this is the way things work, but why do we need to precompute `indexOf()` for each role here? It seems like each of these booleans could be functions that accept `User`, or one function that accepts `User` and `Role`.

Also, why do we need to add a boolean for this role, if a user with this role will never log into the UI? Can we wait to add this until it's needed?

Reply:
We have a test that ensures we can compute the presence of a role on a user.  That role presence is computed in this way, and without this, a test failed.

You raise a great point of if we should be considering doing something else, overall.  I'd like to revisit that in the future.

Reply:
`isAlumni` is a very legacy role - we haven't used that in at least 5 years.